spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -341,10 +346,12 @@ SubscriberStubSettings buildGlobalSubscriberStubSettings() throws IOException {
     SubscriberStubSettings.Builder subscriberStubSettings =
         buildStubSettingsWithoutConfigurations();
 
-    if (this.pullEndpoint != null) {
-      subscriberStubSettings.setEndpoint(this.pullEndpoint);
-    } else {
-      applyGlobalPullEndpoint(subscriberStubSettings);
+    String endpoint =
```

Comment:
Unrelated change for consistency sake, removing the need for `applyGlobalPullEndpoint()`.